code/index.html

```diff
@@ -20,34 +25,56 @@
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
-
+      <h1>Does the doggie have</h1>
+<!--The alternatives to choose from when asking the questions-->
       <select title="questions" id="questions">
-        <optgroup label="hair">
-          <option value="brown">brown hair</option>
-          <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
+        <option value="default" disabled selected >choose an option</option>
```

Comment:
Go team Orange! So happy that we added this part to make the game work more smooth!